docs/src/prototypes/chatPane/data/dateUtils.ts

```diff
@@ -0,0 +1,48 @@
+import * as _ from 'lodash'
+import moment from 'moment'
+import { date, random } from 'faker'
```

Comment:
Remove random if we are not using it, I got this error:
```
[at-loader] Checking finished with 1 errors
[at-loader] ./docs/src/prototypes/chatPane/data/dateUtils.ts:3:16
    TS6133: 'random' is declared but its value is never read.
```

Reply:
thx